CHANGELOG.md

```diff
@@ -248,6 +250,7 @@ in the naming of release branches.
 - `MAClass` is gone: #3175
 - `ShelleyMAEra` type in favor of `AllegraEra` and `MaryEra`: #3175
 - `MATxBody` type in favor of `AllegraTxBody` and `MaryTxBody`: #3175
+- Removed the `TranslateEra` instances for `ShelleyGenesis`: #3164
```

Comment:
There is already an entry for this above, since it they were also removed in #3224 
```suggestion
```